app/javascript/partials/_nested_forms.js

```diff
@@ -89,14 +88,7 @@ $(function() {
     beforeAddForm: ($container) => {
       $container.children('.nested-medium-form').removeClass('d-none');
     },
-    afterAddForm: function(_, $form) {
-
-      // If we only have one nested media form, we don't have to do anything
-      // becasue the index of 0 is already the right one
-      if ($('.nested-medium-form').length === 1) {
-        return;
-      }
```

Comment:
> require('../partials/_fileupload') war die eigentliche Änderung, oder?

@marcometz nein, wie im commit beschrieben war das hier die eigentlich änderung

> - removed code because after deleting all existing file sections and starting over again with adding one, functionality was not bind due to that early return
>   - it was only working again after adding at least a second file section to have a length larger than 1 to bypass the condition

Reply:
Bin mir nicht ganz sicher, aber ich glaube tatsächlich, es müsste eigentlich nur `window.bindFileUploadEvents();` aufgerufen werden, wenn es nur eine Form gibt 🤷 

Aber die Logik ist so komplex und es gibt keine Tests, also wenn es so funktioniert und keine Side-Effects gibt, dann passts wohl. Bin mir allerdings nicht sicher, ob das mit `oldFormIndex` und `formIndex` korrekt ist, wenn man die alten Felder löscht, diesen Fall hatte ich nicht bedacht

Reply:
Hoffe auch, dass ich möglichst viele Varianten manuell getestet habe und auch Julia darüber hinaus. Es schien jetzt soweit gut zu klappen.